src/myuw-notifications.js

```diff
@@ -111,21 +111,105 @@ export class MyUWNotifications extends HTMLElement {
             this event or else the window on click will always fire
             and the menu will never open.
         */
-        this.$bell.addEventListener('click', e => {
-            e.stopPropagation();
```

Comment:
I'd move the clarifying comment about why propagation-stopping is needed closer to where the propagation stopping is now implemented, around line 123.

Reply:
Done!